src/lib/support/TypeTraits.h

```diff
@@ -26,8 +26,21 @@
 
 #include <type_traits>
 
+#if __has_include(<utility>) // For C++23 and later, include <utility> if available
+#include <utility>           // Contains std::to_underlying
+#endif
+
 namespace chip {
 
+#if __cplusplus >= 202300L
+
+template <class T>
+constexpr auto to_underlying(T e)
```

Comment:
Does `using to_underlying = std::to_underlying;` or similar not work?

Reply:
Right. Done